bundles/catalogue/metadatasearch/instance.js

```diff
@@ -0,0 +1,411 @@
+                this.__addTileAndFlyout();
+            }
+        },
+
+        renderSearch: function () {
+            ReactDOM.render(<MetadataSearchContainer state={this.handler.getState()} controller={this.handler.getController()} />, this.contentElement);
+        },
+
+        /* ----------- Tile and Flyout ------------- */
+        __addTileAndFlyout: function () {
+            const request = Oskari.requestBuilder('userinterface.AddExtensionRequest')(this);
+            this.getSandbox().request(this, request);
+            // attach content to flyout when divmanazer has set up root element
+            this.plugins['Oskari.userinterface.Flyout'].getEl().append(this.contentElement);
+        },
+
+        /**
+         * @method startExtension
+         * implements Oskari.userinterface.Extension protocol startExtension method
+         * Creates a flyout and a tile:
+         * Oskari.mapframework.bundle.layerlist.Flyout
+         * Oskari.mapframework.bundle.layerlist.Tile
+         */
+        startExtension: function () {
+            const title = this.getTitle();
+            this.plugins['Oskari.userinterface.Tile'] = Oskari.clazz.create(
+                'Oskari.userinterface.extension.DefaultTile', this, {
+                    title
+                });
+            this.plugins['Oskari.userinterface.Flyout'] = Oskari.clazz.create(
+                'Oskari.userinterface.extension.DefaultFlyout', this, {
+                    title
+                });
+        },
+        /**
+         * @method stopExtension
+         * implements Oskari.userinterface.Extension protocol stopExtension method
+         * Clears references to flyout and tile
+         */
+        stopExtension: function () {
+            this.plugins['Oskari.userinterface.Flyout'] = null;
+            this.plugins['Oskari.userinterface.Tile'] = null;
+        },
+        /**
+         * @method getPlugins
+         * implements Oskari.userinterface.Extension protocol getPlugins method
+         * @return {Object} references to flyout and tile
+         */
+        getPlugins: function () {
+            return this.plugins;
+        },
+        /* ----------- /Tile and Flyout ------------- */
+        /**
+        * @method addSearchResultAction
+        * Add search result action.
+        * @public
+        * @param {jQuery} actionElement jQuery action element
+        * @param {Function} callback the callback function
+        * @param {String} bindCallbackTo the jQuery selector where to bind click operation
+        * @param {String} actionTextElement action text jQuery selector. If it's null then text showed on main element
+        * @param {String} actionText action text
+        * @param {Function} showAction function. If return true then shows action text. Optional.
+        */
+        addSearchResultAction: function (actionElement, actionTextElement, callback, bindCallbackTo, actionText, showAction) {
```

Comment:
I think we can just remove everything regarding search result action in this implementation. If someone is using them we can add them here later. If anyone is using them they will probably need some changes on the usage (jQuery -> React)  as well even if we include it directly.

Reply:
-> Lets remove these.

Reply:
done & done.